Mibi notebook analysis

PozhidayevaDarya opened the pull request.

clarisse-lau commented on HTAN/R Notebooks/Explore_HTAN_Spatial_Cellular_Relationships.Rmd:
````diff
@@ -0,0 +1,313 @@
+```{r}
+df <- read_csv("/Users/vthorsson/HTAN/HMS/coords_keratin.csv",show_col_types = FALSE)
+df <- df %>% rename(Keratin=Keratin_570_cellRingMask,X=X_centroid,Y=Y_centroid)
+df_small <- df %>% filter(X>5000,X<7500,Y>20000,Y<22500)
````
Super minor, but wondering if this code block should be removed?

vthorsson replied:
@clarisse-lau good catch. This has been removed.

clarisse-lau replied:
thank you @vthorsson !

PozhidayevaDarya committed:
Add files via upload

PozhidayevaDarya committed:
Add files via upload

PozhidayevaDarya committed:
Table pulling fix

clarisse-lau commented:
Fix production table name (pull from column C of [HTAN-Tables-metadata](https://docs.google.com/spreadsheets/d/1WWfRCg4LaOZY8EThRTXt34uYE0sbv9VPVf_V1gaWUGM/edit#gid=1661235783) )

PozhidayevaDarya committed:
Fix wrong table name

PozhidayevaDarya committed:
Delete HTAN/Python Notebooks/Analyzing_Higher_Level_HTAN_Imaging_Data.ipynb

PozhidayevaDarya committed:
Addressing Vesteinn's comments

clarisse-lau commented:
Update Open in Colab link to reflect new notebook name

clarisse-lau commented:
Also per @Gibbsdavidl comment on my notebook, let's comment out 
`%env GOOGLE_APPLICATION_CREDENTIALS=<path to key>`

I guess we can assume that most people will use Colab, so commenting this out will allow downstream cells to run smoothly

PozhidayevaDarya committed:
Final updates

Name change in google collab link
Updated note on subsetting in Biospecimen in BQ
Comment out google cred.
